src/components/site-form.tsx

```diff
@@ -338,166 +338,160 @@ export const SiteForm = ( {
 								error={ fileError }
 							/>
 						</div>
-						{ onSelectPath && (
-							<>
-								<div className="flex flex-row items-center mb-0">
-									<Button className="pl-0" onClick={ handleAdvancedSettingsClick }>
-										<Icon
-											size={ 24 }
-											icon={ chevronIcon }
-											className={ error ? 'text-red-500' : '' }
-										/>
-										<div
-											className={ cx(
-												'text-[13px] leading-[16px] ml-2',
-												error ? 'text-red-500' : ''
-											) }
-										>
-											{ __( 'Advanced settings' ) }
-										</div>
-									</Button>
-									{ errorCount > 0 && (
-										<span className="text-red-500 text-[13px] leading-[16px] ml-2 flex items-center">
-											<Icon icon={ warning } size={ 16 } className="mr-1 fill-red-500" />
-											{ sprintf(
-												/* translators: %d: number of errors found */
-												_n( '%d error found', '%d errors found', errorCount ),
-												errorCount
-											) }
-										</span>
-									) }
-								</div>
+					</>
+				) }
+
+				{ onSelectPath && (
+					<>
+						<div className="flex flex-row items-center mb-0">
+							<Button className="pl-0" onClick={ handleAdvancedSettingsClick }>
+								<Icon size={ 24 } icon={ chevronIcon } className={ error ? 'text-red-500' : '' } />
 								<div
-									className={ cx(
-										'transition-all duration-500 ease-in-out overflow-hidden flex flex-col gap-2 interpolate-size-allow-keywords',
-										isAdvancedSettingsVisible ? 'h-auto opacity-100' : 'h-0 opacity-0'
-									) }
+									className={ cx( 'text-[13px] leading-[16px] ml-2', error ? 'text-red-500' : '' ) }
```

Comment:
In `cx`, we don't need to use a ternary condition.

```suggestion
									<div className={ cx( 'text-[13px] leading-[16px] ml-2', error && 'text-red-500' ) }>
```